src/submission/models.py

```diff
@@ -1932,6 +1932,31 @@ def pinned(self):
             return True
 
 
+class FrozenAuthorManager(models.Manager):
+
+    def get_or_create(self, defaults=None, **kwargs):
+        """
+        Backwards-compatible override for affiliation-related kwargs
+        """
+        # check for deprecated fields related to affiliation
+        institution = kwargs.pop('institution', '')
+        department = kwargs.pop('department', '')
+        country = kwargs.pop('country', '')
+
+        frozen_author, created = super().get_or_create(defaults, **kwargs)
+
+        # create or update affiliation
+        if institution or department or country:
+            Affiliation.get_or_create_without_ror(
+                institution=institution,
+                department=department,
+                country=country,
+                frozen_author=frozen_author,
+            )
+
+        return frozen_author, created
+
```

Comment:
as with `Account`, we should be thorough here and make sure we cover the main ORM methods and do so at the `Queryset` level